hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/AbstractAttributeEnricherTest.java

```diff
@@ -178,18 +177,16 @@ protected AttributeValue createAvroAttribute(String value) {
     return AttributeValue.newBuilder().setValue(value).build();
   }
 
-  Event.Builder createOpenSourceSpan(String tenantId, String eventId, String jaegerService) {
-    Map<String, AttributeValue> rawAttributeMap = Map.of(
-        Constants.getRawSpanConstant(JaegerAttribute.JAEGER_ATTRIBUTE_SERVICE_NAME),
-        AttributeValue.newBuilder().setValue(jaegerService).build()
-    );
-
-    Map<String, String> enrichedAttr = Map.of(Constants.getEnrichedSpanConstant(Api.API_BOUNDARY_TYPE), "ENTRY");
+  Event.Builder createOpenSourceSpan(String tenantId, String eventId, String jaegerService,
+      String spanKind) {
```

Comment:
Are you referring here spanType and not SPAN_ATTRIBUTE_SPAN_KIND == span.kind, right? So, let's rename this argument name to String spanType